.github/workflows/dart.yml

```diff
@@ -2,9 +2,9 @@ name: Dart CI
 
 on:
   push:
-    branches: [ master ]
+    branches: [ master, dev ]
```

Comment:
I don't think there is a `dev` branch.

Reply:
But it would be useful.
For example, I wouldn't have to do PR in master)

Reply:
Changing branch structure is out of scope of this PR. Let's keep the concerns separated.